Makefile

```diff
@@ -11,17 +9,22 @@ test:
 test-integration:
 	go version
 	terraform --version
+	. ./tf-dev-override.sh
```

Comment:
Any additional prep here would also need to be added to the docs for running a single integration test.

Reply:
Done